### PR TITLE
Improve binary-only related docs

### PR DESCRIPTION
**Pull request by bet4it**

### CODE REVIEW

File: docs/fuzzing_binary-only_targets.md [hunk 1]
```diff
@@ -302,7 +297,6 @@ some are very hard to set up...
 * S2E: [https://github.com/S2E](https://github.com/S2E)
 * TinyInst:
   [https://github.com/googleprojectzero/TinyInst](https://github.com/googleprojectzero/TinyInst)
-  (Mac/Windows only)
```
Comment (vanhauser-thc) [inline docs/fuzzing_binary-only_targets.md:305]: Why is this removed?

File: docs/fuzzing_binary-only_targets.md [hunk 2]
```diff
@@ -113,10 +112,6 @@ The mode is approximately 2-5x slower than compile-time instrumentation, and is
 less conducive to parallelization. But for binary-only fuzzing, it gives a huge
 speed improvement if it is possible to use.
 
-If you want to fuzz a binary-only library, then you can fuzz it with frida-gum
-via frida_mode/. You will have to write a harness to call the target function in
-the library, use afl-frida.c as a template.
-
```
Comment (vanhauser-thc) [inline docs/fuzzing_binary-only_targets.md:119]: Why is this removed?

**Comment by vanhauser-thc** [comment]
Thank you. Please check out my two questions

File: docs/fuzzing_binary-only_targets.md (hunk 1 above)
Comment (bet4it) [reply docs/fuzzing_binary-only_targets.md:305]: > There is (experimental) Linux support now! :)

_Originally posted by @ifratric in https://github.com/googleprojectzero/TinyInst/issues/67#issuecomment-1593237810_

File: docs/fuzzing_binary-only_targets.md (hunk 2 above)
Comment (bet4it) [reply docs/fuzzing_binary-only_targets.md:119]: https://github.com/AFLplusplus/AFLplusplus/commit/0f299a3bef12b2a8f27a5ba629a2ecb1201fcd60

**Comment by vanhauser-thc** [comment]
Thank you!

**Comment by domenukk** [comment]
Should we add some LibAFL pointers to these docs?

**Comment by vanhauser-thc** [comment]
Sure go for it :)
